src/OrchardCore/OrchardCore.Abstractions/Localization/LocalizationCookieName.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Localization;
+
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Contains a localization cookie names.
+/// </summary>
+public class LocalizationCookieName
```

Comment:
Please remove this unused class, which is now wrong I think.

Reply:
Right